app/javascript/src/LoginPage/RequestPasswordWrapper.jsx

```diff
@@ -0,0 +1,41 @@
+// @flow
```

Comment:
File should be called `RequestPassword.jsx`.

Reply:
@damianpm wdyt ? 🤔

Reply:
We have other files in the same directory with the same 'Wrapper' suffix: `LoginPageWrapper.jsx` and `SignupPageWrapper.jsx` we should keep the same format or change all files.
In the scope of this task, I think we can keep same name, just to not affect files out of scope.